src/lib/pipeline_configuration.py

```diff
@@ -56,11 +62,20 @@ class PipelineConfiguration(object):
                    time_field="capyei_change_time",
                    coda_filename="capyei_change.json",
                    icr_filename="capyei_change.csv",
+                   thematic_analysis_filename="capyei_change_thematic_new.csv",
+                   prev_thematic_analysis_filename="capyei_change_thematic.csv",
                    analysis_file_key="capyei_change_",
                    cleaner=None,
                    code_scheme=CodeSchemes.CHANGE)
     ]
 
+    DEMOGS = [
+        CodingPlan(raw_field="course_name", coded_field=None, coda_filename=None),
+        CodingPlan(raw_field="Sex", coded_field=None, coda_filename=None),
```

Comment:
```suggestion
        CodingPlan(raw_field="sex", coded_field=None, coda_filename=None),
```
???

Reply:
It was inconsistent in the original demogs and have standardised it...